wewatch-client/src/components/Landing/Landing.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { Link } from "react-router-dom";
+
+import './Landing.css';
+
+export default function SignIn() {
+  const [name, setName] = useState('');
+
+  return (
+    <div className="joinOuterContainer">
+      <div className="joinInnerContainer">
+        <h1 className="heading">WeWatch</h1>
+        <div >
+          <input placeholder="Name" className="joinInput" type="text" onChange={(event) => setName(event.target.value)} />
+        </div>
+          {/*TODO: ROUTE HOST USERS TO HOST LOBBY*/}
+          <Link onClick={e => (!name) ? e.preventDefault() : null} to={`/lobby?name=${name}`}>
```

Comment:
I think you achieved already this because of our earlier conversation where we agreed that only guests would have a room/lobby ID in their URL query params. I can see that only the guest flow results in a URL with the lobby ID query param so we can remove this TODO for now